src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/BrowseObjectBase.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.AttachedColl
     /// right from it)
     /// </summary>
     [ComVisible(true)]
-    internal abstract class BrowseObjectBase : ICustomTypeDescriptor
+    public abstract class BrowseObjectBase : ICustomTypeDescriptor
```

Comment:
Does this actually need to be public and COMVisible? This is typically only needed for IDispatch to work, but ICustomTypeDescriptor is skipping CLR's IDispatch.

Reply:
`ComVisible` is not needed. Removed.